connectors/bigquery/flink-connector-gcp-bigquery/src/test/java/com/google/flink/connector/gcp/bigquery/BigQueryTypeUtilsTest.java

```diff
@@ -0,0 +1,219 @@
+package com.google.flink.connector.gcp.bigquery;
+
+import com.google.api.services.bigquery.model.TableFieldSchema;
+import org.apache.flink.table.api.DataTypes;
+import org.apache.flink.table.types.DataType;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static com.google.common.truth.Truth.assertThat;
+
+/** Tests for {@link BigQueryTypeUtils}. */
+public class BigQueryTypeUtilsTest {
+
+  @Test
+  public void testToFlinkTypeString() {
```

Comment:
you can condense this and the below tests (to testToFlinkTypeBignumeric) into a single parameterized test